geopandas/geodataframe.py

```diff
@@ -600,21 +600,59 @@ def to_parquet(self, filename, compression="snappy", index=None, **kwargs):
         Parameters
         ----------
         path : str, path object
+        index : bool, default None
+            If ``True``, always include the dataframe's index(es) as columns
+            in the file output.
+            If ``False``, the index(es) will not be written to the file.
+            If ``None``, the index(ex) will be included as columns in the file
+            output except `RangeIndex` which is stored as metadata only.
         compression : {'snappy', 'gzip', 'brotli', None}, default 'snappy'
             Name of the compression to use. Use ``None`` for no compression.
+        kwargs
+            Additional keyword arguments passed to to pyarrow.parquet.write_table().
+        """
+
+        from geopandas.io.arrow import _to_parquet
+
+        _to_parquet(self, path, compression=compression, index=index, **kwargs)
+
+    def to_feather(self, path, index=None, **kwargs):
+        """Write a GeoDataFrame to the Feather format.
+
+        Any geometry columns present are serialized to WKB format in the file.
+
+        Requires 'pyarrow' >= 0.17.
+
+        WARNING: this is an initial implementation of Feather file support and
+        associated metadata.  This is tracking version 0.1.0 of the metadata
+        specification at:
+        https://github.com/geopandas/geo-arrow-spec
+
+        This metadata specification does not yet make stability promises.  As such,
+        we do not yet recommend using this in a production setting unless you are
+        able to rewrite your Feather files.
+
+        .. versionadded:: 0.8
+
+        Parameters
+        ----------
+        path : str, path object
         index : bool, default None
             If ``True``, always include the dataframe's index(es) as columns
             in the file output.
             If ``False``, the index(es) will not be written to the file.
             If ``None``, the index(ex) will be included as columns in the file
             output except `RangeIndex` which is stored as metadata only.
+        compression : {'zstd', 'lz4', 'uncompressed'}, optional
```

Comment:
Apologies for not catching this sooner, but if `compression` is in the docstring it should be in the function parameters too, with the default value:

```
def to_feather(self, path, index=None, compression=None, **kwargs):
```

Also please add default value (and missing `None`) to the docstring, same as for `to_parquet`
```
compression : {'zstd', 'lz4', 'uncompressed', None}, default None.
```

(here and `_to_feather` also)

Reply:
> Also please add default value (and missing None) to the docstring, same as for to_parquet

It's a a bit a semantic discussion ;), but the "optional" can also be seen as already mentioning the default of None (in general I think you will see `, default None` and `, optional` quite mixed in projects)

Reply:
Sure...  Mostly I was noticing the inconsistency of how this is handled between `to_parquet` and `to_feather` and also `index` parameter immediately above.